cactus/page.py

```diff
@@ -75,16 +75,11 @@ def context(self, data=None, extra=None):
         """
```

Comment:
Sure, I had left it in because somebody else might rely on them being there. But that seems overly cautious.